modules/test/playwright/tests/commerce/commerce-product-definitions-web/commerceAdminProductVisibility.spec.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {commercePagesTest} from '../../../fixtures/commercePagesTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import getRandomString from '../../../utils/getRandomString';
+import performLogin, {performLogout} from '../../../utils/performLogin';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	loginTest()
+);
+
+test('LPD-25206 Admin product page shows correct account groups', async ({
+	apiHelpers,
+	commerceAdminProductDetailsPage,
+	commerceAdminProductDetailsVisibilityPage,
+	commerceAdminProductPage,
+	page,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount({
```

Comment:
can we add the same test but using the admin to verify that he is able to view ALL the account groups?